language-ruby_5bb9e082042863158cc7236d.md

```diff
@@ -92,6 +93,81 @@ Semaphore doesn't set language specific environment variables like
             - bundle exec rake test
 </code></pre>
 
+## Rails
+
+You'll need to set `RAILS_ENV` and configure a databse. Rails 4.1+
+supports using the `DATABASE_URL` to [configure a
+database][rails-database-configuration]. Older versions of Rails need
+to generate a `database/config.yml` manually.
+
+### 4.1+ - Using DATABASE_URL
+
+Rails will read database configuration from `DATABASE_URL` if set and
+`config/database.yml` is empty. So, start a database with
+[sem-service][sem-service], set `DATABASE_URL`, and remove the
+existing `config/database.yml`. Here's an example block:
```

Comment:
Let's just add a note that this is optional and applies if you have a `config/database.yml` in Git.

Reply:
It's more accurate to say if your `config/database.yml` is not blank. My guess is the majority of cases have `config/database.yml` in SCM (since it's the rails default) and you need the file to configure dev and test bases. It seems that using `DATABASE_URL` is primarily for deployments (or CI in our case).

Reply:
Agreed.